client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -203,7 +203,14 @@ class DisconnectJetpack extends PureComponent {
 		);
 	};
 
-	handleTryRewind = () => this.props.trackTryRewind( this.props.siteSlug );
+	handleTryRewind = () => {
+		this.props.recordTracksEvent( 'calypso_disconnect_jetpack_try_rewind' );
```

Comment:
I think this needs to be `recordTracksEventAction` (either that, or L212 is incorrect 😅).